<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageSetDialog</class>
 <widget class="QDialog" name="ImageSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>575</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1030</width>
    <height>575</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1030</width>
    <height>575</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Image Set erstellen</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="videoLabelLeft">
         <property name="minimumSize">
          <size>
           <width>480</width>
           <height>320</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>480</width>
           <height>320</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>3</number>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="videoLabelRight">
         <property name="minimumSize">
          <size>
           <width>480</width>
           <height>320</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>480</width>
           <height>320</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>3</number>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="showVideosCheckBox">
       <property name="text">
        <string>Videos anzeigen</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="numberImages_label">
             <property name="text">
              <string>Anzahl  Bilder:</string>
             </property>
             <property name="buddy">
              <cstring>numberImages_spinBox</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="rows_label">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>1</number>
             </property>
             <property name="midLineWidth">
              <number>0</number>
             </property>
             <property name="text">
              <string>Anzahl Reihen:</string>
             </property>
             <property name="buddy">
              <cstring>rows_spinBox</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="columns_label">
             <property name="text">
              <string>Anzahl Spalten:</string>
             </property>
             <property name="buddy">
              <cstring>columns_spinBox</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="squareSize_label">
             <property name="text">
              <string>Kastengröße:</string>
             </property>
             <property name="buddy">
              <cstring>squareSize_SpinBox</cstring>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QSpinBox" name="numberImages_spinBox">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="minimum">
              <number>10</number>
             </property>
             <property name="maximum">
              <number>50</number>
             </property>
             <property name="value">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="rows_spinBox">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="maximum">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="columns_spinBox">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="maximum">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="squareSize_SpinBox">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="minimum">
              <double>1.000000000000000</double>
             </property>
             <property name="maximum">
              <double>99.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="pattern_label">
             <property name="text">
              <string>Welches Pattern:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="chessboard_radioButton">
             <property name="text">
              <string>Schachbrett</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="circle_radioButton">
             <property name="text">
              <string>Kreise</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="asymCircles_radioButton">
             <property name="text">
              <string>Kreise asymmetrisch</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="setImageSet_checkBox">
           <property name="text">
            <string>Als Standard für Kalibrierunng festlegen</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="imageSetRecord_button">
           <property name="text">
            <string>Aufnahme starten</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
